lib/simulation/openfisca/mapping/index.js

```diff
@@ -16,31 +16,26 @@ var isNotValidValue = function(value) {
 var getPeriods = function(dateDeValeur) {
     dateDeValeur = moment(dateDeValeur).freeze();
     return {
-        moisCourant: {
+        thisMonth: {
             key: dateDeValeur.format('YYYY-MM')
```

Comment:
No, let's keep it consistent. It should be changed everywhere if we simplify it here.